### PR TITLE
Debugdraw enhancements

### PR DESCRIPTION
**Pull request by metalgearsloth**
AKA Sloth copy-pastes from box2d (yes I will add the licence when I'm done). Already helped me spot some physics serialization issues that would've stopped predicted pushing.

### CODE REVIEW

File: Robust.Client/Console/Commands/PhysicsOverlayCommands.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@ namespace Robust.Client.Console.Commands
     public sealed class PhysicsOverlayCommands : IConsoleCommand
     {
         public string Command => "physics";
-        public string Description => $"{Command} <contactnormals / contactpoints / shapes>";
-        public string Help => $"{Command} <overlay>";
+        public string Description => $"Shows a debug physics overlay. The arg supplied specifis the overlay.";
```
Comment (PaulRitter) [inline Robust.Client/Console/Commands/PhysicsOverlayCommands.cs]:
```suggestion
        public string Description => $"Shows a debug physics overlay. The arg supplied specifies the overlay.";
```

File: Robust.Client/Graphics/Clyde/Clyde.RenderHandle.cs [hunk 2]
```diff
@@ -389,9 +389,9 @@ public override void DrawCircle(Vector2 position, float radius, Color color, boo
                             _renderHandle.DrawLine(startPos, endPos, color);
                         else
                         {
-                            filledTriangle[0] = startPos;
-                            filledTriangle[1] = endPos;
-                            filledTriangle[2] = Vector2.Zero;
+                            filledTriangle[0] = startPos + position;
+                            filledTriangle[1] = endPos + position;
+                            filledTriangle[2] = Vector2.Zero + position;
```
Comment (PaulRitter) [inline Robust.Client/Graphics/Clyde/Clyde.RenderHandle.cs:394]: this bug hits different ngl

**Review by PaulRitter — APPROVED** [review]
lgtm, just a nitpick

File: Robust.Client/Graphics/Clyde/Clyde.RenderHandle.cs (hunk 2 above)
Comment (metalgearsloth) [reply Robust.Client/Graphics/Clyde/Clyde.RenderHandle.cs:394]: image

File: Robust.Client/Debugging/DebugPhysicsSystem.cs [hunk 3]
```diff
@@ -108,59 +141,210 @@ internal struct ContactPoint
...
+        private readonly Font _font;
+
+        public PhysicsDebugOverlay(IEyeManager eyeManager, IInputManager inputManager, DebugPhysicsSystem system, SharedBroadphaseSystem broadphaseSystem)
         {
+            _eyeManager = eyeManager;
+            _inputManager = inputManager;
             _physics = system;
+            _broadphaseSystem = broadphaseSystem;
+            var cache = IoCManager.Resolve<IResourceCache>();
+            _font = new VectorFont(cache.GetResource<FontResource>("/Fonts/NotoSans/NotoSans-Regular.ttf"), 10);
```
Comment (metalgearsloth) [inline Robust.Client/Debugging/DebugPhysicsSystem.cs]: Woops forgot this unga bad will fix before merge

**Pull request merged**